deployment/network-operator/templates/operator.yaml

```diff
@@ -27,8 +30,15 @@ spec:
       {{- include "network-operator.selectorLabels" . | nindent 6 }}
   template:
     metadata:
+      {{- if .Values.operator.admissionController.enabled }}
+      annotations:
+        kubectl.kubernetes.io/default-container: {{ .Chart.Name }}
+      {{- end }}
       labels:
         nvidia.com/ofed-driver-upgrade-drain.skip: "true"
+        {{- if .Values.operator.admissionController.enabled }}
```

Comment:
same, we can always add this one.  WDYT ?

Reply:
done